src/sgf_parser/models/method.py

```diff
@@ -52,16 +52,34 @@ def comment_code_format(cls, data: Any) -> Any:
         Sometimes we get more than one code on a data row, then we need to prioritize what code to keep.
         """
         if isinstance(data, dict):
-            # Detect string variants of the comment code and convert them to integers
             if "K" in data and data["K"] is not None:
-                _code, _rest = cls._extract_comment_code(data)
-                data["K"] = _code
-                if _rest:
+                
+                # We want to interpret K as a stop code (with integer value)
+                # sometimes K is a string with i.e. "SAND", in that case we want to move it to T
+                
+                data_K_split = data["K"].split(", ") # in case there are multiple K codes
+                K_is_digit = all([x[0:2].isdigit() for x in data_K_split]) 
```

Comment:
what if its 1 number instead of 2 ? is it always 2? add a guard for checking number in k string